README.md

```diff
@@ -24,6 +24,24 @@ __If you are still curious about Whim__, by all means check it out and [give me
 
 You may also wish to join the [\#whim channel on Freenode IRC](http://webchat.freenode.net/?channels=%23whim), where I am likely idling but listening as `jmac`.
 
+## Building and testing whim locally
+
+[App::cpanminus]: https://metacpan.org/pod/App::cpanminus
+
+Use the CPAN or [App::cpanminus][] tools to install dependencies.
+
+``` shell-session
+$ cpanm --installdeps --with-develop .
+```
+
+And `prove` for the tests!
+
+``` shell-session
+$ prove t/ xt/
```

Comment:
Should this be `prove -l t/ xt/`? Without the `-l` it doesn't know to add `lib/` to @INC, does it?

Reply:
It works without it because of the `use lib` pragmas. However, using `-l` is more idiomatic and doesn't break anything, so :+1:!